arwen/contexts/metering.go

```diff
@@ -193,20 +193,7 @@ func (context *meteringContext) TrackGasUsedByBuiltinFunction(err error, builtin
 		return err
```

Comment:
it is no sense to return the error back. just send the error and that's it.

Reply:
Removed `return err` and added a `log.Trace()`.